src/api/bucket_api.js

```diff
@@ -781,6 +781,74 @@ module.exports = {
                 system: ['admin', 'user']
             }
         },
+
+        put_bucket_logging: {
+            method: 'PUT',
+            params: {
+                type: 'object',
+                required: ['source_bucket', 'log_bucket', 'log_prefix'],
+                properties: {
+                    source_bucket: {
+                        $ref: 'common_api#/definitions/bucket_name'
+                    },
+                    log_bucket: {
+                        $ref: 'common_api#/definitions/bucket_name'
+                    },
+                    log_prefix: {
+                        type: 'string',
+                    },
+                },
+                // auth: {
```

Comment:
why is the auth commented out? this property specifies which roles are allowed to perform the operation, by removing it the operation can be performed unauthenticated